parser/parser.go

```diff
@@ -355,14 +331,47 @@ func (p *parser) evaluateOperator(value1, value2 Expression, operator rune,
 			switch v := value.(type) {
 			case *String:
 				v.Value += e2.(*String).Value
+			case *Int64:
+				v.Value += e2.(*Int64).Value
```

Comment:
It occurs to me that even though we currently support the "+" operator for lists, and we're now adding the ability to use integers, we don't necessarily have to support the "+" operator for integers too, which might be considered to be too much logic.

(And there's also the thought that even if we do disallow the "+" operator for lists, maybe that won't be enforced at the parser level but instead somewhere else)

Maybe we want to return an error and wait until there's a use case.

Maybe Dan or Colin will offer an opinion

Reply:
I can't think of a case where + would be useful, but I also can't think of any other semantics we would want besides this and the implementation is trivial so I'm fine either way.